apps/fluent-tester/src/FluentTester/TestComponents/ButtonExperimental/ButtonSizeTestSection.tsx

```diff
@@ -0,0 +1,28 @@
+import { Button, CompoundButton } from '@fluentui-react-native/experimental-button';
+import { SvgIconProps } from '@fluentui-react-native/icon';
+import * as React from 'react';
+import { Platform, View } from 'react-native';
+import { commonTestStyles, stackStyle } from '../Common/styles';
+import TestSvg from './test.svg';
+
+export const ButtonSizeTest: React.FunctionComponent = () => {
+  const svgProps: SvgIconProps = {
+    src: TestSvg,
+    viewBox: '0 0 500 500',
+  };
+  const svgIconsEnabled = ['ios', 'macos', 'win32', 'android'].includes(Platform.OS as string);
+
+  return (
+    <View style={[stackStyle, commonTestStyles.view]}>
+      {svgIconsEnabled ? <Button size="small" icon={{ svgSource: svgProps }} style={commonTestStyles.vmargin} /> : null}
```

Comment:
Should here be 3 conditions for svgIconsEnabled? Can it be one? And Button components can be in react fragment.

Reply:
Yeah, this can be simplified. Thanks for catching!

Reply:
Did some cleanup